core/frontend/src/components/parameter-editor/ParameterEditor.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <v-card
+    width="700"
```

Comment:
I believe that editing the width is not the way to go, makes the screen really empty, only the margin should be enough.
With limited width:
![image](https://user-images.githubusercontent.com/1215497/199753353-b49cde13-cbe5-431e-98d3-082f6a123c09.png)

Without:
![image](https://user-images.githubusercontent.com/1215497/199752554-a53cc7e9-6018-44cf-b73b-217853d4215c.png)

Reply:
yep, that looks better